containers-ordered-strict/src/Map/Ord.hs

```diff
@@ -4,26 +4,26 @@ module Map.Ord
        ( Map
        , Key
 
-       , empty
-       , singleton
-       , fromList
-
-       , null
-       , size
-       , member
-       , lookup
+       , M.empty
+       , M.singleton
```

Comment:
Okay, looks like `singleton` also should not be exported. I see the following error on the Travis CI:

```haskell
    • Identifier ‘singleton’ has conflicting definitions in the module
      and its hsig file
      Main module: singleton ::
                     k -> a -> containers-0.5.11.0:Data.Map.Internal.Map k a
      Hsig file:  singleton :: Key k => k -> v -> Map k v
      The two types are different
```